README.md

```diff
@@ -19,17 +19,17 @@ Requirements
 
 * PHP 7+
 
-How to install
+How to install Woketo:
```

Comment:
It's a title, so no `:`. Also it's quite obvious that we are talking about woketo, isn't it? :) (maybe some people will install it without being aware that there's a name to this library)